unit_tests/test_lib_charms_ovn_charm.py

```diff
@@ -564,6 +568,7 @@ def setUp(self):
                 'br-provider:00:01:02:03:04:05 br-other:eth5'),
             'ovn-bridge-mappings': (
                 'provider:br-provider other:br-other'),
+            'prefer-chassis-as-gw': True,
```

Comment:
Should we assert somewhere that `opvs.set('.', 'external_ids:ovn-cms-options', 'enable-chassis-as-gw')` gets called?

Reply:
Yes we do that on https://github.com/openstack-charmers/charm-layer-ovn/blob/9b47e2e0f519e33ee222e31d63e75f41a0cb8e56/unit_tests/test_lib_charms_ovn_charm.py#L849-L854